### PR TITLE
Update description of gomnd

### PR DESCRIPTION
**Pull request by tommy-muehle**
The previous description was a bit confusing. Because there is not "the" magic number. So I changed it to a more clearer version in my opinion.

### CODE REVIEW

File: pkg/golinters/gomnd.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ func NewGoMND() *goanalysis.Linter {
 
 	return goanalysis.NewLinter(
 		"gomnd",
-		"checks whether magic number is used",
+		"A analyzer to detect magic numbers.",
```
Comment (bombsimon) [inline pkg/golinters/gomnd.go]:
```suggestion
		"An analyzer to detect magic numbers.",
```

Comment (tommy-muehle) [reply pkg/golinters/gomnd.go]: Good catch! Thanks.

**Comment by tpounds** [comment]
@tommy-muehle Can you run `make README.md` to regenerate a new README with the updated description?

**Comment by tommy-muehle** [comment]
@tpounds Done :)

**Comment by tpounds** [comment]
@tommy-muehle Thanks! Can you also sing the CLA?

**Comment by tommy-muehle** [comment]
@tpounds Also done :)

**Comment by tpounds** [comment]
@tommy-muehle Thanks! :rocket:

**Pull request merged**